### DISCUSSION TITLE
Issues Connecting to Secure Server using Python SDK

### DISCUSSION THREAD

**Discussion by oscarramirez-bobit** [ravendb/ravendb#19600] [Q&A]
I am trying to query a collection using the Python SDK but cannot connect using the generated client certificate. The end goal is to create a data pipeline to Azure blob storage. Can anyone share how they connect to a secured server with the Python SDK or if they recommend another approach to this pipeline.

**Comment by ayende** [comment]
Did you see the docs here? 
https://github.com/ravendb/ravendb-python-client/?tab=readme-ov-file#working-with-secured-server

What is the _exact_ error (including stack trace, etc) that you are getting after following this guide?
